Unable to login when enable-AzPSSession is enabled

When I try logging in with `enable-AzPSSession: true` I get the following output:

```
Legal docs and information: aka.ms/AzureCliLegal


You have 2 updates available. Consider updating your CLI installation with 'https://aka.ms/installazurecliwindows'. Detailed instructions can be found at https://aka.ms/doc/UpdateAzureCliMsi

Please let us know how we are doing: https://aka.ms/clihats
and let us know if you're interested in trying out our newest features: https://aka.ms/CLIUXstudy
Running Azure PS Login
C:\Program Files\PowerShell\7\pwsh.exe -Command "try ***
            $ErrorActionPreference = \"Stop\"
            $WarningPreference = \"SilentlyContinue\"
            $output = @***
            $data = Get-Module -Name Az.Accounts -ListAvailable | Sort-Object Version -Descending | Select-Object -First 1
            $output['AzVersion'] = $data.Version.ToString()
            $output['Success'] = \"true\"
        ***
        catch ***
            $output['Error'] = $_.exception.Message
        ***
        return ConvertTo-Json $output"
##[error]Azure PowerShell Login failed. Please check the credentials. For more information refer https://aka.ms/create-secrets-for-GitHub-workflows"
##[error]Error: There was an error when attempting to execute the process 'C:\Program'. This may indicate the process failed to start. Error: spawn C:\Program ENOENT
```

However, if I set `enable-AzPSSession: false` it logs in fine. 

Note: I'm using `windows-latest` as the runner.

--- a/lib/PowerShell/Utilities/PowerShellToolRunner.js
+++ b/lib/PowerShell/Utilities/PowerShellToolRunner.js
@@ -28,7 +28,7 @@ class PowerShellToolRunner {
     }
     static executePowerShellScriptBlock(scriptBlock, options = {}) {
         return __awaiter(this, void 0, void 0, function* () {
-            yield exec.exec(`${PowerShellToolRunner.psPath} -Command`, [scriptBlock], options);
+            yield exec.exec(`"${PowerShellToolRunner.psPath}" -Command`, [scriptBlock], options);
         });
     }
 }

--- a/src/PowerShell/Utilities/PowerShellToolRunner.ts
+++ b/src/PowerShell/Utilities/PowerShellToolRunner.ts
@@ -11,6 +11,6 @@ export default class PowerShellToolRunner {
     }
 
     static async executePowerShellScriptBlock(scriptBlock: string, options: any = {}) {
-        await exec.exec(`${PowerShellToolRunner.psPath} -Command`, [scriptBlock], options)
+        await exec.exec(`"${PowerShellToolRunner.psPath}" -Command`, [scriptBlock], options)
     }
 }

